source/installation_guide/upgrading/from-2.3-to-3.0.rst

```diff
@@ -260,3 +260,11 @@ And re-configure the ACL plugin:
 Afterwards you can remove the old global ACL directory parent::
 
    rm -rf /etc/dovecot/acls/
+
+
+Changes to statistics
+---------------------
+
+ - The bytes\_in and bytes\_out fields have been renamed as net\_in\_bytes and net\_out\_bytes. 
```

Comment:
\`\`bytes_in\`\` with quotes? Also, why do you need to escape _?

Reply:
reflex, i think my mind was writing markdown